pkgs/top-level/all-packages.nix

```diff
@@ -21447,7 +21447,7 @@ with pkgs;
     icu74
   ;
 
-  icu = icu73;
+  icu = icu74;
```

Comment:
A mere change of defaults? Well, let's wait ofBorg then.

Reply:
Seems like all checks [except for tests on x86_64-darwin](https://github.com/NixOS/nixpkgs/pull/290509/checks?check_run_id=25692459832) have passed, not sure if these are related.

Reply:
ARM Darwin passed.